src/main/java/org/motechproject/newebodac/mapper/VaccineeMapper.java

```diff
@@ -19,16 +18,16 @@
 import org.motechproject.newebodac.helper.EncryptionHelper;
 import org.springframework.beans.factory.annotation.Autowired;
 
-@Mapper(uses = { ExtraFieldMapper.class }, unmappedTargetPolicy = ReportingPolicy.IGNORE)
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+@Mapper(uses = { ExtraFieldMapper.class },
+    componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.IGNORE)
 public abstract class VaccineeMapper implements EntityMapper<VaccineeDto, Vaccinee> {
-  
-  public static final VaccineeMapper INSTANCE = Mappers.getMapper(VaccineeMapper.class);
 
   @Autowired
   private EncryptionHelper encryptionHelper;
 
-  @Mapping(target = "phoneNumber",
-      qualifiedByName = "ToEncryptedPhoneNumber")
+  @Mapping(target = "phoneNumber", qualifiedByName = "ToEncryptedPhoneNumber")
```

Comment:
you can add a static field with the "ToEncryptedPhoneNumber" and "ToDecryptedPhoneNumber" to avoid suppressing the pmd